tests/factories.py

```diff
@@ -193,10 +193,26 @@ def update_fulltext(obj, create, extracted, **kwargs):
 
 
 class FilingsFactory(BaseFactory):
+    sub_id = factory.Sequence(lambda n: n)
+
     class Meta:
         model = models.Filings
 
 
+class EFilingsFactory(BaseFactory):
+    file_number = factory.Sequence(lambda n: n)
+
+    class Meta:
+        model = models.EFilings
+
+
+class BaseFilingFactory(BaseFactory):
+    file_number = factory.Sequence(lambda n: n)
+
+class BaseF3PFilingFactory(BaseFilingFactory):
```

Comment:
Might be good to add factories for the other base classes for efilings.  But we can take that one for the issue of adding tests for efilings.